_docs/10_general/yourgdpr.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Your GDPR using Simple Analytics
+category: general
+category_order: 1
+order: 10
+permalink: /your-gdpr
+---
+## Should you update your PP if you install our analytics solution on your website?
+
+We are a privacy first product, which is why we made everything necessary to be able to respond to this with a simple "No you don't have to".
+
+Indeed, officially there is no need to add us to your Privacy Policy because Simple Analytics doesn't store IP adresses or any other personal identifiable information.
+
+We also don’t use cookies, so we don’t need to be added to your cookie banner either.
+For more information on what we collect(and mostly what we don't) here is the related documentation page: https://simpleanalytics.com/what-we-collect
```

Comment:
Links don't get clickable automatically and we have what-we-collect on docs as well: https://docs.simpleanalytics.com/what-we-collect. Maybe use

```
For more information on what we collect (and mostly what we don't) [go to what we collect](https://docs.simpleanalytics.com/what-we-collect).
```